repo/meta/index.json

```diff
@@ -260,6 +260,22 @@
       }
     },
     {
+      "currentVersion":"5.7.12",
+      "description":"MySQL is the world's most popular open source database. With its proven performance, reliability and ease-of-use, MySQL has become the leading database choice for web-based applications, covering the entire range from personal projects and websites, via e-commerce and information services, all the way to high profile web properties including Facebook, Twitter, YouTube, Yahoo! and many more.",
+      "framework":true,
+      "name":"mysql",
+      "selected":true,
+      "tags":[
+        "mesosphere",
+        "framework",
```

Comment:
tags are used for end users to search.  the word mesosphere doesn't make sense.  if allowed it would result in all services being returned.   we are trying to remove all references to the word "framework" as well.